templates/ingress.yaml

```diff
@@ -61,6 +55,11 @@ spec:
               number: 8080
         path: /sockets
         pathType: Prefix
-  {{- if ((.Values.ingress).tls).secretName }}
+      - backend:
+          service:
+            name: {{ .Release.Name }}-frontend-svc
+            port:
+              number: 8080
+        path: /
+        pathType: Prefix
     host: {{ .Values.domain }}
```

Comment:
I'm not sure if we should set host here? We still want to support the case where customers have multiple frontend hostnames which I think should be the default. If a customer needs to limit things to a single domain (separately from the TLS settings) that should be opt in